skyportal/handlers/api/instrument.py

```diff
@@ -230,10 +233,17 @@ def put(self, instrument_id):
         data['id'] = int(instrument_id)
 
         # permission check
-        _ = Instrument.get_if_accessible_by(
+        instrument = Instrument.get_if_accessible_by(
```

Comment:
Should `raise_if_none` be replaced with a check on `instrument is None` and a `self.error`?

Reply:
yes.